packages/console/src/pages/ApiResourceDetails/index.tsx

```diff
@@ -46,43 +39,14 @@ const ApiResourceDetails = () => {
   const theme = useTheme();
   const Icon = theme === AppearanceMode.LightMode ? ApiResource : ApiResourceDark;
 
+  const isOnPermissionTab = pathname.endsWith(ApiResourceTabs.Permissions);
```

Comment:
Should we use the route params to define those paths? And then using `useParams` to get the tab key. End with looks hacky.

Reply:
updated.

Reply:
updated with nested routes.